pkgs/tools/misc/copier/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib, gnused, poetry, git, python3, fetchFromGitHub }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "copier";
+  version = "unstable-2021-11-10";
```

Comment:
FWIW [Copier 6 was recently released](https://github.com/copier-org/copier/releases/tag/v6.0.0). You'd probably prefer to package that?